### ISSUE TITLE
Node version issue

### ISSUE THREAD

**Issue by agrohs** [nrkno/nodecache-as-promised#11]
We receive an error whenever we try to use this package in a node 10 environment of the following `error @nrk/nodecache-as-promised@1.3.2: The engine "node" is incompatible with this module. Expected version "^8.7.0". Got "10.16.3"` (where 10.16.3 is whatever version is running for that app build).

It looks like in the `package.json` file, you have it fixed to node 8 w/ the following:
```
  "engines": {
    "node": "^8.7.0",
    "npm": "^5.4.0"
  }
```

why not just change that to something similar to:
```
  "engines": {
    "node": ">=8.x",
    "npm": "^5.4.0"
  }
```

**Comment by agrohs** [comment]
See PR #12

**Comment by agrohs** [comment]
Looks like this was fixed with release of version 1.3.4!!
